src/hotspot/share/runtime/java.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1997, 2021, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1997, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
You can revert the copyright header change since there's no other changes in this file.